assets/src/edit-story/components/canvas/canvasLayout.js

```diff
@@ -1,62 +1,223 @@
 /**
  * External dependencies
  */
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 /**
  * Internal dependencies
  */
-import { CENTRAL_RIGHT_PADDING, LEFT_NAV_WIDTH, PAGE_WIDTH, PAGE_HEIGHT, PAGE_NAV_PADDING } from '../../constants';
-import Page from './page';
+import { LEFT_NAV_WIDTH, PAGE_WIDTH, PAGE_HEIGHT, PAGE_NAV_BUTTON_WIDTH } from '../../constants';
+import { useStory } from '../../app';
+import useCanvas from './useCanvas';
+import PageDisplay from './page';
+import PageFrames from './pageFrames';
+import PageEdit from './pageEdit';
 import PageMenu from './pagemenu';
 import PageNav from './pagenav';
 import Carrousel from './carrousel';
 import SelectionCanvas from './selectionCanvas';
 
-const Background = styled.div`
+// QQQQ: remove.
+const DEBUG_ANGLE = 0;
+
+const WorkspaceBackground = styled.div`
 	background-color: ${ ( { theme } ) => theme.colors.bg.v1 };
-	display: flex;
-	flex-direction: column;
-	align-items: center;
-	justify-content: center;
+	width: 100%;
 	height: 100%;
+	position: relative;
+	user-select: none;
+
+	${ DEBUG_ANGLE && ( `
+		background-color: darkgray;
+		backface-visibility: hidden;
+		transform-style: preserve-3d;
+		transform-origin: 50% 50%;
+		transform: perspective(2500px) scale(0.4) rotateY(${ DEBUG_ANGLE }deg);
+	` ) }
+`;
+
+const Layer = css`
```

Comment:
"Position" instead of Layer, maybe?

Reply:
also would prefer to make variable names more explicit plus lower case as they're not components (styled components docs also use lowercase), e.g. `positionCss` or `positionCSS`.

Reply:
Still prefer "Layer" or "CanvasLayer" since they do not reflect positions of any kind - only `z-index` layering. This will soon stop being "css" and become an actual component - still deciding what would look better here.

Reply:
sure, if it becomes a component then works for me.